Sources/Interaction/Manipulators/KeyboardCameraManipulator/index.js

```diff
@@ -225,7 +248,7 @@ function vtkKeyboardCameraManipulator(publicAPI, model) {
 // ----------------------------------------------------------------------------
 
 const DEFAULT_VALUES = {
-  movementSpeed: 0.02,
+  movementSpeed: null,
```

Comment:
maybe you could add a comment to explain the meaning of "null" (or 0) and what the unit is. (e.g. length travelled by the camera at each frame each 1/60 second)

Reply:
Done. And actually, I also changed it to be an explicit check for `null` (as in, if the `movementSpeed === null`, then the `movementSpeed` gets reset when movement starts).